src/pages/Ranking/style.ts

```
@@ -0,0 +1,78 @@
+  flex-direction: column;
+  justify-content: center;
+  gap: 24px;
+  padding: 0 32px;
+  width: 100%;
+  max-width: 1180px;
+  height: 146px;
+  border-radius: 8px;
+  box-shadow: 0px 4px 20px rgba(179, 180, 184, 0.2);
+`;
+
+export const Date = styled.div`
```

Biome (1.9.4)

[error] Do not shadow the global "Date" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)